homepage.js

```diff
@@ -0,0 +1,11 @@
+let total = 0;
+
+function addToCart () {
+  total = 123.45 + total;
+  alert(total)
+}
+
+
+function dogImage (dogName, dogBreed, dogCost) {
+  alert (dogName + ", " + dogBreed + ", " + dogCost)
```

Comment:
Very Good! You might be interested in template literals as a way to concatenate text and variables. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals